### DISCUSSION TITLE
Quick-Clear for text fields like document title

### DISCUSSION THREAD

**Discussion by thoughtgap** [LeoKlaus/Paperparrot#190] [Feature Requests]
I tried using Paperparrot for inbox-clearing, and the UX is a bit cumbersome. Every document is filled with the filename which I usually replace with the actual document title.

Problem: The document title field does not have a quick-clear button (usually an `(X)` on the right side of the form field) to remove all contents. So currently, I need to mark the whole field's content and remove it or overwrite it directly.

A quick-clear button in the form field would speed up this process.

**Comment by LeoKlaus** [comment]
There was already a quick clear button while uploading, I've now added one for all title fields. It might take a while before this hits the AppStore, I wan't to make some other changes with the next update as well.
